website/docs/guides/docs/sidebar/autogenerated.md

```diff
@@ -194,6 +194,98 @@ Naming your introductory document `README.md` makes it show up when browsing the
 
 :::
 
+### Customizing category index matching
```

Comment:
IMHO this doc should be much smaller and less exhaustive

The goal is not to have 1000 words for each tiny customization feature we have, otherwise we are distracting users from reading the essential information (and the metadata seems more important than this section). 

Maybe hide all this in a summary/details view?